app/components/c-linklist.md

```diff
@@ -0,0 +1,6 @@
+### Styles
+
+#### Linklist
+```
+<c-linklist :items="[{ key: 'Montageanleitung als PDF', value: 'LinkTo: Montageanleitung als PDF' }, { key: 'Produktdatenblatt', value: 'LinkTo: Produktdatenblatt' }, { key: 'Zertifikate', value: 'LinkTo: Zertifikate' }]"/>
+```
```

Comment:
[ ] Add examples for all 3 props

Reply:
Do you mean links to actually existing documents?